crates/matrix-sdk-base/src/rooms/normal.rs

```diff
@@ -578,6 +589,11 @@ impl Room {
         Ok(members)
     }
 
+    /// Get the heroes for this room.
+    pub fn heroes(&self) -> Vec<RoomHero> {
+        self.inner.read().heroes().to_owned()
```

Comment:
Why `.to_owned()` here? If it is because the underlying return type is `&[...]`, can you use `to_vec()` to make this more obvious?

Reply:
Oh, so unlike `&str` where `to_owned` is preferred over `to_string()` for arrays `to_vec` is preferred over `to_owned`?

Reply:
Hmm good question — I don't know about Clippy (who may disagree with me), but having `to_vec()` makes it more visible that it was a slice, because that's one of the few structs that has a `to_vec()` on it.